src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -8074,7 +8193,14 @@ protected override string Dump(LocalState state)
                 if (nameForSlot(i) is string name)
                 {
                     builder.Append(name);
-                    builder.Append(state[i] == NullableFlowState.MaybeNull ? "?" : "!");
+                    var annotation = (state[i]) switch
+                    {
+                        NullableFlowState.MaybeNull => "?",
+                        NullableFlowState.MaybeDefault => "??",
```

Comment:
>"??" [](start = 58, length = 4)

Will this be confused for two consecutive slots each MaybeNull?

Reply:
It wasn't confusing because we print variable names and annotations. So it shows `x?y??z!` for example